<?xml version="1.6" ?>
<launch>
	
    <!-- PARAMETERS -->
    <arg name="motor_node" value="left_side_wheels" />
    <arg name="sub_topic" value="leftCommand" />

    <arg name="serial_back_port" value="/dev/ttyUSB0" />
    <arg name="serial_front_port" value="/dev/ttyUSB1" />

    <arg name="robot_side_value" value="left" />

    <arg name="serial_baud_value" value="57600" />
    <arg name="serial_parity_value" value="0" /> <!-- 0:None, 1:odd, 2:even -->
    <arg name="serial_stop_bit_value" value="1" /> <!-- 1:One, 2:Two, 3:One point five -->
    <arg name="serial_data_bits_value" value="8" /> <!-- 5, 6, 7 or 8 -->
    <arg name="serial_hdw_flow_ctrl_value" value="0" /> <!-- 0:None, 1:software, 2:hardware -->
    <arg name="serial_timeout_value" value="400" />

    <include file="$(find engrais_motors)/launch/motor.launch">
	    <arg name="motor_node_name" value="$(arg motor_node)" />
	    <arg name="sub_topic_name" value="$(arg sub_topic)" />

	    <arg name="back_port" value="$(arg serial_back_port)" />
	    <arg name="front_port" value="$(arg serial_front_port)" />

	    <arg name="robot_side" value="$(arg robot_side_value)" />

	    <arg name="serial_baud" value="$(arg serial_baud_value)" />
	    <arg name="serial_parity" value="$(arg serial_parity_value)" />
	    <arg name="serial_stop_bit" value="$(arg serial_stop_bit_value)" />
	    <arg name="serial_data_bits" value="$(arg serial_data_bits_value)" />
	    <arg name="serial_hdw_flow_ctrl" value="$(arg serial_hdw_flow_ctrl_value)" />

	    <arg name="serial_timeout" value="$(arg serial_timeout_value)" />

    </include>

</launch>
